### PR TITLE
feat(public cloud project dashboard): add products and new links in project dashboard EU/CA

### PR DESCRIPTION
**Pull request by baaastijn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-8410
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

The public cloud universe added lot of products and services but not the dashboard.
Added more buttons, with graphical style

- Before : Instance, volume, container OS, private network, kubernetes
- After : Instances, Kubernetes, volume, container OS, Databases, AI notebooks, PRivate network, Failover IP

Why these 8 : it's the top8 of clicks in the left menu

Images : took official website images, validated with OVH CX their use

Documentation Links : validated internally with the Steerco PCI
Community :Links : same

## Local tests : Firefox 92 MacOS ans Safari
used basic bootstrap rules

Desktop view :
pr1

Smartphone :
pr2
pr3

Signed-off-by: Bastien Verdebout (bastien.verdebout at ovhcloud dot com)

### CODE REVIEW

**Comment by baaastijn** [comment]
added status and signed-off my 2 commits

File: packages/manager/modules/pci/src/projects/project/project.html [hunk 1]
```diff
@@ -61,18 +61,20 @@
                             <div class="container-fluid px-0">
                                 <div class="row">
                                     <div
-                                        class="col-sm-6 col-lg-4 col-xl mb-4"
+                                        class="d-inline-block col-sm-6 col-lg-4 col-xl-3 mb-4"
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/project.html:64]:
```suggestion
                                        class="col-sm-6 col-md-4 col-lg-3 mb-4"
```

File: packages/manager/modules/pci/src/projects/project/project.html [hunk 2]
```diff
@@ -61,18 +61,20 @@
...
                                             class="text-center w-100 d-block oui-tile py-4"
                                             data-ui-state="action.state"
                                             data-ui-state-params="{projectId: $ctrl.project.project_id}"
                                         >
-                                            <span
-                                                class="oui-icon oui-icon-add"
-                                                aria-hidden="true"
-                                            ></span>
+                                            <img
+                                                class="inline-block"
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/project.html:73]:
```suggestion
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/project.html [hunk 3]
```diff
@@ -99,14 +100,21 @@
                                         </ul>
                                     </div>
                                     <div class="col-sm-6 mb-4">
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/project.html:102]:
```suggestion
                                    <div
                                        class="col-sm-6 mb-4"
                                        data-ng-if="$ctrl.coreConfig.getRegion() !== 'US'"
                                    >
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/project.html (hunk 3 above)
Comment (baaastijn) [reply packages/manager/modules/pci/src/projects/project/project.html]: taken into account int he last commit

**Review by antleblanc — APPROVED** [review]

**Pull request merged**